.env

```diff
@@ -0,0 +1 @@
+REACT_APP_DOMAIN="http://localhost:4000"
```

Comment:
I don't think we want to share the .env file on github, please add .env to the .gitignore :)

Reply:
Fixed!